Radzen.Blazor/RadzenDropDown.razor

```diff
@@ -152,7 +152,7 @@
                             <input id="@SearchID" tabindex="@(Disabled ? "-1" : $"{TabIndex}")" class="rz-inputtext" role="textbox" type="text"
                                    onclick="Radzen.preventDefaultAndStopPropagation(event)" aria-label="@SearchAriaLabel"
                                    @ref="@search" @oninput=@(args => { searchText = $"{args.Value}"; SearchTextChanged.InvokeAsync(searchText);})
-                                   @onchange="@((args) => OnFilter(args))" @onkeydown="@((args) => OnFilterKeyPress(args))" value="@searchText" autocomplete="@FilterAutoCompleteType" />
```

Comment:
We will not merge this for sure since it will change the behavior completeely.

Reply:
> and causes it to trigger on close of the filter/popup
Never seen such problem - feel free to point us to some of our demos where this can be reproduced:
https://blazor.radzen.com/dropdown-filtering?theme=material3

Reply:
Ah, I see now what you are after - I missunderstood the problem completely, sorry! Changing however onchange to onkeydown will brake FilterAsYouType which can be set to false (true by default). I'll do my best to fix both problems befor the end of the week.